Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: cli
+cli:
+	npm install
+	npm run build
+	mkdir -p cli/pkg/web/embed/dist
+	cp -r web-local/build/ cli/pkg/web/embed/dist
+	go build -ldflags="-X 'main.Version=2.0.0' -X 'main.DocsUrl=https://docs.rilldata.com'" -o rill cli/main.go 
```

Comment:
The docs URL should just be a constant in the code, not passed here

For now, you also don't need to pass the version here, just set it to `"dev"` by default in the code. When we set up goreleaser, it can inject version and commit sha, see example here:       - -s -w -X main.version={{.Version}} -X main.commit={{.ShortCommit}} -X main.date={{.Date}}